src/test/scala/com/microsoft/hyperspace/index/HybridScanGeneralTest.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index
+
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.execution.datasources._
+
+import com.microsoft.hyperspace.Hyperspace
+
+class HybridScanGeneralTest extends HybridScanTestSuite {
```

Comment:
Maybe `HybridScanForNonPartitionedDataSuite` and `HybridScanForPartitionedDataSuite`, if I understand the separation correctly? (A small description would also help)